restonomer-core/src/main/scala/com/clairvoyant/restonomer/core/transformation/RestonomerTransformation.scala

```diff
@@ -82,3 +82,13 @@ case class ReplaceStringInColumnValue(
     restonomerResponseDF.replaceStringInColumnValue(columnName, pattern, replacement)
 
 }
+
+case class AddPrefixToColumnNames(
+    prefix: String,
+    columnNames: List[String] = null
```

Comment:
Don't initialise it as null. Initialise it as an empty list.

Reply:
Done! instead of using Null values, now I am using empty list object.